backend/src/apis/IotaConnector.test.ts

```diff
@@ -0,0 +1,59 @@
+import { IndexationPayload } from '@iota/client/lib/types'
+
+import { sendDataMessage, getMessage, getAllMessages } from '@/apis/IotaConnector'
+import { CONFIG } from '@/config'
+
+jest.mock('@/config', () => ({
+  CONFIG: {
+    IOTA: true,
+    IOTA_COMMUNITY_ALIAS: 'GRADIDO: TestHelloWelt1',
+    IOTA_API_URL: 'https://chrysalis-nodes.iota.org',
+  },
+}))
+
+describe('apis/IotaConnector/enabled', () => {
+  describe('Hello World', () => {
+    const now = new Date()
+    let messageId = ''
+    const messageString = 'Hello World - ' + now.toString()
+    const messageHexString = Buffer.from(messageString, 'utf8').toString('hex')
+    const indexHexString = Buffer.from(CONFIG.IOTA_COMMUNITY_ALIAS, 'utf8').toString('hex')
+    it('send hello world message to iota tangle', async () => {
+      const iotaMessage = await sendDataMessage(messageString)
+      expect(iotaMessage).not.toBeNull()
+      if (iotaMessage) {
```

Comment:
Please no if statements in unit tests. Otherwise you would have to test the test